src/java/org/apache/cassandra/index/sai/view/IndexViewManager.java

```diff
@@ -28,34 +28,40 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import org.apache.cassandra.index.sai.IndexContext;
 import org.apache.cassandra.index.sai.IndexValidation;
 import org.apache.cassandra.index.sai.SSTableContext;
+import org.apache.cassandra.index.sai.StorageAttachedIndex;
 import org.apache.cassandra.index.sai.disk.SSTableIndex;
 import org.apache.cassandra.index.sai.StorageAttachedIndexGroup;
+import org.apache.cassandra.index.sai.utils.IndexIdentifier;
+import org.apache.cassandra.index.sai.utils.IndexTermType;
 import org.apache.cassandra.io.sstable.format.SSTableReader;
 import org.apache.cassandra.utils.Pair;
 
 /**
  * Maintain an atomic view for read requests, so that requests can read all data during concurrent compactions.
- *
+ * <p>
  * All per-column {@link SSTableIndex} updates should be proxied by {@link StorageAttachedIndexGroup} to make
  * sure per-sstable {@link SSTableContext} are in-sync.
  */
 public class IndexViewManager
 {
     private static final Logger logger = LoggerFactory.getLogger(IndexViewManager.class);
     
-    private final IndexContext context;
+    private final StorageAttachedIndex index;
+    private final IndexTermType indexTermType;
+    private final IndexIdentifier indexIdentifier;
```

Comment:
nit: Could we get by w/ having just the `index` field, given it contains both the term type and the identifier?